### PR TITLE
Time sync robustness

### PR DESCRIPTION
**Pull request by maseabunikie**
This change requires to enable network=host in timesync-hypervisor 
```
        {
            "Name": "timesync-hypervisor",
            "Repository": "https://github.com/tiiuae/dockerfiles",
            "Maintainers": "joonas.loppi@unikie.com",
            "Revision": "c00127142e626cd3d7a34cc9bdc8881607f52922",
            "ManualEditingPreventionChecksum": "f1",
            "Artifacts": [
                {
                    "ImageRef": "ghcr.io/tiiuae/tii-chrony:sha-c001271",
                    "amd64": {
                        "ImageRefByDigest": "ghcr.io/tiiuae/tii-chrony@sha256:de304683495858f13de1f73ba0b9826351d9542d2c38da15625a68b9b1a17d26",
                        "Digest": "2c3afd5a03cdf8c8750f6f1c3cde90911360aefa6bef56f6cfe5b23242483b04",
                        "Size": 14155143
                    },
                    "arm64": {
                        "ImageRefByDigest": "ghcr.io/tiiuae/tii-chrony@sha256:e63671f33a4a326199d43303569f5d409be7b9aa0eae929ff71885ba416220a4",
                        "Digest": "21184b8c7feca8638ff88b827f5887d83184b364c43744ce9dd4056d4c4ac269"
                    },
                    "riscv64": {
                        "ImageRefByDigest": "ghcr.io/tiiuae/tii-chrony@sha256:89c6e270d0fa7b7c681e20b85b226e44831fbe86e016fd59c9bdb620702debfe",
                        "Digest": "08fe29c461dca53c8c2e6c8e6526babc1fdbf456f1d4500ddeb7799bf8bafd9d"
                    }
                }
            ],
            "Notes": "Synchronizes time from public NTP to hypervisor. rtcsync asks the kernel to sync system time to RTC every 11 minutes.",
            "UserGuide": "https://github.com/tiiuae/fog_hyper/tree/main/docs/time",
            "ContainerOptions": {
                "RunsOn": {
                    "Default": "$hypervisor"
                },
                "DockerOpts": [
                    "--network=host",
                    "--env=CHRONY_CONF=pool time.cloudflare.com iburst\\nmakestep 0.1 3\\nrtcsync",
                    "--cap-add=SYS_TIME"
                ],
                "DockerOptsAdditional": {}
            }
        },
```

### CODE REVIEW

File: debs/amd64/chronyc_4.2-r0_amd64.deb
Comment (mehmetkillioglu) [inline debs/amd64/chronyc_4.2-r0_amd64.deb:1]: Seems like the artifacts from local debugging pushed to the branch also. Those are available in artifactory

Comment (maseabunikie) [reply debs/amd64/chronyc_4.2-r0_amd64.deb:1]: Removed.

**Comment by maseabunikie** [comment]
After a chat with @mehmetkillioglu , it has been agreed to change the behavior when the time sync is checked. So by default it will be the same as before. If the CHRONY_DAEMON_IP address is set in the environment then the time sync will be checked.

**Review by mehmetkillioglu — APPROVED** [review]

**Review by jnippula — APPROVED** [review]

**Comment by maseabunikie** [comment]
Chrony is not used anymore in SalukiV3 + fog-ghaf. This change is not valid anymore.

**Pull request closed without merging**